.github/workflows/feature.yml

```diff
@@ -0,0 +1,41 @@
+---
+
+name: Feature Branch Image Build and Push
+
+on:
+  workflow_dispatch:
+
+  push:
+    branches: [feature_*]
```

Comment:
@TheRealHaoLiu Since you've already done the work here, should we just go ahead and build and push images on pushes to the `devel` branch as well?

Reply:
out of scope for now